sources/presentation/Xenko.Core.Presentation/Themes/ExpressionDark/Theme.xaml

```diff
@@ -2299,39 +2302,56 @@
-              <TextBlock x:Name="IGTTextBlock" Grid.Column="3" Text="{TemplateBinding MenuItem.InputGestureText}" IsEnabled="{TemplateBinding IsEnabled}" Margin="{TemplateBinding Control.Padding}" VerticalAlignment="Center" Visibility="Visible"/>
+              <ContentPresenter Grid.Column="1" x:Name="HeaderHost" RecognizesAccessKey="True" ContentSource="Header" Margin="0,3" VerticalAlignment="Center" HorizontalAlignment="Stretch" SnapsToDevicePixels="True" />
+              <TextBlock x:Name="IGTTextBlock" Grid.Column="3" Text="{TemplateBinding MenuItem.InputGestureText}" IsEnabled="{TemplateBinding IsEnabled}" Margin="{TemplateBinding Control.Padding}" VerticalAlignment="Center" Visibility="Visible" Padding="0,3" />
               <Grid Grid.Column="4" Margin="4,2" x:Name="ArrowPanel" VerticalAlignment="Center">
                 <Path x:Name="ArrowPanelPath" VerticalAlignment="Center" Fill="{TemplateBinding Foreground}" Data="M0,0 L0,8 L4,4 z" />
               </Grid>
               <Popup IsOpen="{Binding IsSubmenuOpen, RelativeSource={RelativeSource TemplatedParent}}" Placement="Right" x:Name="SubMenuPopup" Focusable="false" AllowsTransparency="true" PopupAnimation="{DynamicResource {x:Static SystemParameters.MenuPopupAnimationKey}}">
-                <Grid x:Name="SubMenu">
-                  <Border x:Name="SubMenuBorder" Background="{DynamicResource MenuBackgroundBrush}" BorderBrush="{DynamicResource MenuBorderBrush}" BorderThickness="1"/>
-                  <StackPanel IsItemsHost="True" KeyboardNavigation.DirectionalNavigation="Cycle" Grid.IsSharedSizeScope="true"/>
+                <Grid>
+                  <Grid.RowDefinitions>
+                    <RowDefinition Height="3" />
+                    <RowDefinition Height="*" />
+                    <RowDefinition Height="3" />
+                  </Grid.RowDefinitions>
+                  <Grid.ColumnDefinitions>
+                    <ColumnDefinition Width="1" />
+                    <ColumnDefinition Width="*" />
+                    <ColumnDefinition Width="1" />
+                  </Grid.ColumnDefinitions>
+                  <Border Grid.Row="0" Grid.ColumnSpan="3" Background="{StaticResource MenuBackgroundBrush}" />
+                  <Border Grid.Row="2" Grid.ColumnSpan="3" Background="{StaticResource MenuBackgroundBrush}" />
+                  <Border x:Name="SubmenuMainWrapperBorder" Grid.RowSpan="3" Grid.ColumnSpan="3" Background="{StaticResource MenuBackgroundBrush}" BorderBrush="{StaticResource MenuBorderBrush}" BorderThickness="0.5" />
+                  <Grid x:Name="SubMenu" Grid.Row="1" Grid.Column="1">
+                    <Border x:Name="SubMenuBorder" Background="{DynamicResource MenuBackgroundBrush}" BorderBrush="{DynamicResource MenuBorderBrush}" BorderThickness="0" />
+                    <StackPanel IsItemsHost="True" KeyboardNavigation.DirectionalNavigation="Cycle" Grid.IsSharedSizeScope="true" />
+                  </Grid>
```

Comment:
Something here is breaking the "view options" menu in the asset view: text and icon are incorrectly aligned (horizontally). But I can't figure exactly where the issue is: Margin, Border, Padding or a combination or something else (the issue could be a side-effect with the style of that menu in the AssetViewUserControl.xaml).

![image](https://user-images.githubusercontent.com/3006525/43900155-f49a1f98-9c1e-11e8-883f-f7b2876ada2c.png)
![image](https://user-images.githubusercontent.com/3006525/43900169-fcda0092-9c1e-11e8-9bcb-42b738960361.png)


What was the reason for changing the submenu template here?

Reply:
Humm... I'll take a look.

Reply:
Weird but, it looks fine in my side 🤔 

![View Options](https://raw.githubusercontent.com/0xaryan/MediaMan/master/Github/PullRequests/EditorUI/Review/ViewOptions.png)

Reply:
Hmm. Really weird. I cleaned the whole solution and recompiled and now the issue is also gone on my side.

Maybe some XAML files were not recompiled correctly and it was mixing the new styles with some remains of the old one.

Reply:
@Kryptos-FR I saw a lot of weird behaviors from visual studio when I was working on this project, specially when working with Edit-and-Continue. Sometimes I need to clean whole solution even for a small change to take effect.